include/swift/AST/Availability.h

```diff
@@ -329,6 +329,11 @@ class AvailabilityContext {
   bool isAvailableAsSPI() const {
     return SPI && *SPI;
   }
+
+  /// Returns a representation of this range as a string for debugging purposes.
+  std::string getAsString() const {
+    return "AvailabilityContext(" + OSVersion.getAsString() + (isAvailableAsSPI() ? ", spi" : "") + ")";
```

Comment:
Minor: run `git-clang-format`

Reply:
Thanks, I'll follow up separately if that's fine? That'd just be a linebreak here AFAICS.

```diff
-    return "AvailabilityContext(" + OSVersion.getAsString() + (isAvailableAsSPI() ? ", spi" : "") + ")";
+    return "AvailabilityContext(" + OSVersion.getAsString() +
+           (isAvailableAsSPI() ? ", spi" : "") + ")";
```

// The problem with clang-format is that it is not being applied consistently in this project, so applying it adds all kinds of noise and "this commit does not look like the rest of the file or method at all now..."

Reply:
Follow up is fine with me! I find the `clang-format` output is pretty consistent with the style of most of the files I have worked in in the project and it is documented in https://github.com/apple/swift/blob/main/docs/HowToGuides/FirstPullRequest.md as the primary way of enforcing the project code style. In particular, long lines stick out a lot - I think the majority of the codebase adheres to the 80 char limit

Reply:
I'll follow up about my specific changes, but the reality is that files are not well formatted: 

```
 22 files changed, 65 insertions(+), 65 deletions(-)
```

if running clang-format on files this PR touched, out of that my changes would be...

```
 5 files changed, 22 insertions(+), 15 deletions(-)
```

but yeah, I'll add a git hook to git-clang-format my commits and I'll do the promised follow up now as well